versions.yaml

```diff
@@ -161,8 +161,8 @@ assets:
     description: "Linux kernel optimised for virtual machines"
     url: "https://cdn.kernel.org/pub/linux/kernel/v4.x/"
```

Comment:
you should update this url, right?

Reply:
yep: https://cdn.kernel.org/pub/linux/kernel/v5.x/
otherwise my (somewhere pending, maybe merged) 'version check scripts' will fail to find which versions are available, and probably complain loudly.

Reply:
Yup. I am holding off any changes to this patch and CI churn until we land kernel fragments change. Once that is done, i can update this patch and also PR the kernel patches update in packaging repo.